api/openapi.yaml

```diff
@@ -3,12 +3,12 @@ info:
   contact:
     email: support@cloudsmith.io
     name: Cloudsmith Support
-    url: https://help.cloudsmith.io
+    url: https://support.cloudsmith.com/hc/en-us
```

Comment:
@BartoszBlizniak can you confirm this is the right contact link for the support team?